docs/en/quickstart/rest_api.md

```diff
@@ -84,7 +84,11 @@ The request body example:
 ```json
 {
     "mode": "online",
-    "sql": "select 1"
+    "sql": "SELECT c1, c2, c3 FROM demo WHERE c1 = ? AND c2 = ?",

```

Comment:
add a new example. Query can do normal sql and parameterized sql, do not mislead users to do only parameterized sql